Source/AwsCommonRuntimeKit/http/HttpClientConnectionManager.swift

```diff
@@ -64,21 +73,23 @@ public class HttpClientConnectionManager {
                 return
             }
             let callbackData = userData.assumingMemoryBound(to: HttpClientConnectionCallbackData.self)
-            defer {callbackData.deinitializeAndDeallocate()}
+            defer {
+                callbackData.deinitializeAndDeallocate()
+            }
```

Comment:
Revert lines in this PR that are just style or whitespace changes.

They makes this change hard to review.
And they makes the git history sloppy. On BIG projects like this, someone will inevitably be like "why is this line doing what it's doing" and then go dig through the git history. They're going to see a PR with an interesting description, look at the PR, and be like "dammit that was misleading this is just an arbitrary whitespace change", and it makes the dig harder.

If this code was being auto-formatted before, let's get our auto-formatter configured the same way.

If this code was NOT being auto-formatted, please turn your auto-formatter off for now.

Let's do a separate followup PR where we run an auto-formatter on EVERYTHING and then we can one git commit where a TON of code changes, but the line history will clearly show that this commit was "Autoformat all the code".

This is what we do in nearly all our other repos. Autoformat the code, and have a CI check that enforces it, so that lines aren't arbitrarily changing all the time, and so we're not wasting time nitpicking whitespace style in code reviews

Reply:
Thank you for the detailed explanation. I have fixed it.